grift/tests/test_config.py

```diff
@@ -1,5 +1,6 @@
 # Copyright 2017 Kensho Technologies, Inc.
 import os
+import six
```

Comment:
`six` is pip-installed, so it goes in the second import block. The first one is for built-in libraries only.